applications/Chat/coati/dataset/utils.py

```diff
@@ -20,3 +20,21 @@ def jload(f, mode="r"):
     jdict = json.load(f)
     f.close()
     return jdict
+
+
+def read_data_by_schema(data, schema: str):
```

Comment:
Please add type hint for return value

Reply:
And for all arguments

Reply:
It seems that this function is only used in `read_string_by_schema`. If so, why not merge these two functions?